### DISCUSSION TITLE
Abort after accept() func while running echo binary

### DISCUSSION THREAD

**Discussion by ghost** [ANLAB-KAIST/KENSv3#105] [Project Q&A 1. Application Layer (Socket)] [answered]
I try to test my code with ./echo-non-kens s 0.0.0.0 9000 hello-client and ./echo  test but it stop at accept func with no message.
![image](https://user-images.githubusercontent.com/113764328/190848667-927d8794-c446-4a90-bef8-90a3ebfeb116.png)

how can i fix this?

**Comment by bongjunj** [comment, marked as answer]
accept can return negative integers other than -1

**Reply by ghost** [reply]
Thanks!
